src/openrct2/localisation/string_ids.h

```diff
@@ -3665,6 +3665,9 @@ enum {
 	STR_UNIT1DP_SUFFIX_FEET = 6011,
 	STR_UNIT1DP_NO_SUFFIX = 6012,
 
```

Comment:
Light suggestion:
Remove spacing (_most of the text isn't spaced as a standard, and if it's going to be a norm, then I presume it's ok_)